### PR TITLE
Clicker logic

### PR DESCRIPTION
**Pull request by Sukh-123**

### CODE REVIEW

File: Assets/Scripts/Clicker.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+public class Clicker : MonoBehaviour
+{
+  
+    public int power = 0;
+    public float powerPerSecond = 1.0f;
+    public TMP_Text powerText;
+
+    void Start()
```
Comment (txj0925) [inline Assets/Scripts/Clicker.cs]: Unused method

File: Assets/Scripts/afkBoost.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            autoHarvestTimer -= Time.deltaTime;
+            if (autoHarvestTimer <= 0)
+            {
+                autoHarvestActive = false;
+                HarvestPower();
+            }
+        }
+    }
+
+    public void MainButtonPress()
```
Comment (txj0925) [inline Assets/Scripts/afkBoost.cs]: Method repeated from clicker.cs is it really usefull?

**Comment by Sukh-123** [comment]
yeah sorry the afkBoost class is useless and is not needed

________________________________

File: Assets/Scripts/afkBoost.cs (hunk 2 above)
Comment (Sukh-123) [reply Assets/Scripts/afkBoost.cs]: this not needed

File: Assets/Scripts/Clicker.cs (hunk 1 above)
Comment (Sukh-123) [reply Assets/Scripts/Clicker.cs]: powerPerSecond is needed for OfflineProgress

**Review by txj0925 — APPROVED** [review]
I had to make some changes. We shall review together if everything works as intended

**Review by txj0925 — COMMENTED** [review]
Don't forget to do unit testing next time.

**Pull request merged**